include/exadg/grid/grid.h

```diff
@@ -45,19 +45,6 @@ class Grid
     dealii::GridTools::PeriodicFacePair<typename dealii::Triangulation<dim>::cell_iterator>>
     PeriodicFacePairs;
 
-  /**
-   * Constructor.
-   */
-  Grid()
-  {
-  }
-
-  /**
-   * Initialize function
-   */
-  void
-  initialize(GridData const & data, MPI_Comm const & mpi_comm);
```

Comment:
for full flexibility, `Grid` should not depend on `GridData`, which is achieved with the present PR.